starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -2499,35 +2509,25 @@ sub time2date {
 }
 
 ###################################################################################
-sub count_good_stars{
+sub count_guide_stars{
 ###################################################################################
     my $self=shift;
     my $c;
-    my $clean_acq_count = 0;
-    my $clean_gui_count = 0;
-    $self->{count_nowarn_stars}{ACQ} = $clean_acq_count;
-    $self->{count_nowarn_stars}{GUI} = $clean_gui_count;
+    my $gui_count = 0;
 
     return unless ($c = find_command($self, 'MP_STARCAT'));
     for my $i (1 .. 16){
-        my $type = $c->{"TYPE$i"};
-        next if ($type eq 'NUL');
-        next if ($type eq 'FID');
-	if ($type =~ /ACQ|BOT/){
-	    unless ($self->check_idx_warn($i)){
-		$clean_acq_count++;
-	    }
-	}
-	if ($type =~ /GUI|BOT/){
-	    unless ($self->check_idx_warn($i)){
-		$clean_gui_count++;
-	    }
+	if ($c->{"TYPE$i"} =~ /GUI|BOT/){
+            my $mag = $c->{"MAG$i"};
+            # Use the fractional magnitudes from ORViewer.  tab-ternary is if/elsif/else
+            my $star_contrib = $mag <= 10.0  ? 1
+                             : $mag <= 10.2  ?  .5
```

Comment:
Numbers like `0.5` should always be written with the leading `0` for clarity.  In this context you should go further and write `1.0, 0.75, 0.5, and 0.0` for consistency and clarity that these are fractional star values.

Reply:
Good point.  My alignment game was not sufficient.